### ISSUE TITLE
Motion PDF: Images cannot be scaled

### ISSUE THREAD

**Issue by tsiegleauq** [OpenSlides/OpenSlides#3180]
With the implementation of CKEditor, `<img>` tags in motions got a different set of attributes.
TinyMCE used normale image attributes, CKEditor now uses inline CSS style attributes like:

`style="height: 20px; width: 20px;"`

We need to alter image parsing to allow scaling images in PDFs

@emanuelschuetze , please assign me for this issue

**Comment by tsiegleauq** [comment]
That one's kinda odd.
Maybe somebody else got an idea.
The issue is, every `element.clientWidth` returns `0`

That's extremly odd. Usually, asking an image element `clientWidth` should return the width. Maybe some other code interferes with that... e.g. webworkers?

**Comment by tsiegleauq** [comment]
We can get the size of the base64 image using an async function. This will uglify the code a lot, so I rather not do that now.

```
newImageObj = new Image();
newImageObj.src = base64String;
newImageObj.onload {
  console.log(this.width); //will return the correct width of original image
};
```

**Comment by FinnStutzenstein** [comment]
@tsiegleauq This can be closed now?

**Comment by tsiegleauq** [comment]
Yes. I forgot to add the fixes #3180 in my pull request for this issue. can
be closed now
